hostports/watcher.go

```diff
@@ -109,6 +111,25 @@ func (p PortRule) natIptables() []byte {
 	return buf.Bytes()
 }
 
+func (p ChainRule) filterIptables() []byte {
```

Comment:
In the existing IPSec catalog, if we change the bridge from docker0 to rancher0, things don't work. So may be having these rules in general will help that case too?